client/my-sites/post-selector/docs/example.jsx

```diff
@@ -17,10 +17,17 @@ const PostSelectorExample = React.createClass( {
 
 	getInitialState() {
 		return {
-			showTypeLabels: true
+			showTypeLabels: true,
+			selectedPostID: '',
```

Comment:
This should be camel-cased `selectedPostId`, per [naming guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions).